### PR TITLE
Update keys swagger to include CKM_AES_KEY_WRAP and CKM_AES_KEY_WRAP_PAD for key wrapping

### PR DESCRIPTION
**Pull request by hrjami**
# Data Plane API - Pull Request

- Update keys swagger to include CKM_AES_KEY_WRAP and CKM_AES_KEY_WRAP_PAD for key wrapping. This is only to list these in our swagger. The actual support for these are already there.
- Changed retry header type to int32 in backuprestore

Is this review for (select one):

-  a public preview

### Change Scope

<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous API Spec document (if applicable), and the root paths that have been updated. </sup>
This is not a new change, rather just listing in our swagger a functionality that already exists

* Design Document:
* Previous API Spec Doc:
* Updated paths:

### Viewing API changes

For convenient view of the API changes made by this PR, refer to the URLs provided in the table 
in the `Generated ApiView` comment added to this PR. You can use ApiView to show API versions diff. 

### Suppressing failures

If one or multiple validation error/warning suppression(s) is detected in your PR, please follow the 
[Swagger-Suppression-Process](https://aka.ms/pr-suppressions) 
to get approval.

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azsdk/onboarding/restapischedule) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azsdk/onboarding/restapischedule).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling

 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://github.com/Azure/azure-api-style-guide/blob/main/README.md)

### Guidelines & Specifications

 * [Azure REST API Guidelines](https://aka.ms/azapi/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/azapi/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links

 * [Schedule a data plane REST API spec review](https://aka.ms/azsdk/onboarding/restapischedule)

</details>

## Getting help

- First, please carefully read through this PR description, from top to bottom.
- If you don't have permissions to remove or add labels to the PR, request `write access` per [aka.ms/azsdk/access#request-access-to-rest-api-or-sdk-repositories](https://aka.ms/azsdk/access#request-access-to-rest-api-or-sdk-repositories) 
- To understand what you must do next to merge this PR, see the `Next Steps to Merge` comment. It will appear within few minutes of submitting this PR and will continue to be up-to-date with current PR state.
- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- If the PR CI checks appear to be stuck in `queued` state, please add a comment with contents `/azp run`.
  This should result in a new comment denoting a `PR validation pipeline` has started and the checks should be updated after few minutes.
- If the help provided by the previous points is not enough, post to https://aka.ms/azsdk/support/specreview-channel and link to this PR.

### CODE REVIEW

**Comment by heaths** [comment]
@hrjami in the future, you should create a branch and not work directly in main even from your fork. That's going to create problems when you later try to sync your `main` from the upstream `main`.

**Review by heaths — APPROVED** [review]
@JeffreyRichter @mikekistler the breaking change is from a swagger correction. It's already in the surface. Could you add the label or is it fine for me to in this case?

**Comment by mikekistler** [comment]
@Heaths Anyone can add the "BreakingChange-Approved-BugFix" label, at least for now. This is a bit of an experiment in "self approval" but so far it seems to be going well.

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
               "Retry-After": {
                 "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
                 "type": "integer",
-                "format": "int64"
+                "format": "int32"
```
Comment (JonathanCrd) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json:116]: Thanks for making this change.

I wonder, is it necessary to specify the format for this header? 
I see on [lines 58-61](https://github.com/Azure/azure-rest-api-specs/pull/30115/files#diff-4c203dfce12178b8c7e6ba4dda9615a0d4ae836981d2ea76f84cc40362037ec7R58-R61), where the same header has been defined, the format is not specified.

**Comment by hrjami** [comment]
> @hrjami in the future, you should create a branch and not work directly in main even from your fork. That's going to create problems when you later try to sync your `main` from the upstream `main`.

Addes PublishToCustomers label

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json (hunk 1 above)
Comment (heaths) [reply specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json:116]: It's better to be explicit. Most of the `Retry-After` headers herein are.

**Pull request merged**